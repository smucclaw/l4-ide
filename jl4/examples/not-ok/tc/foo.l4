IMPORT bar

#CHECK res
#EVAL bingus 1 2 3
