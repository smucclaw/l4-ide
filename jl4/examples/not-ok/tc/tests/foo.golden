Parsing successful
Typechecking successful
Info | updateFileDiagnostics published different from new diagnostics - file diagnostics:
File: foo.l4
  Hidden:   no
  Range:    1:1-1:1
  Source:   jl4
  Severity: DiagnosticSeverity_Error
  Code:     <none>
  Message: 
    One of your dependencies depends on this module and forms a cycle: foo.l4 bar.l4 qux.l4 foo.l4
